setup/operators/operators.go

```diff
@@ -24,13 +24,11 @@ const (
 
 var Templates = []string{
 	"devspaces.yaml",
-	"rhoda.yaml",
-	"rhods.yaml",
```

Comment:
Reminder to keep rhods as discussed in meeting

Reply:
done see [d8724f9](https://github.com/codeready-toolchain/toolchain-e2e/pull/675/commits/d8724f9b0bae5839e89d1a07880ebaf3939bf759) 

Sorry for the confusion!

Reply:
the rhods template file needs to be restored as well

Reply:
Good point! 🤦  

Restored in [f022835](https://github.com/codeready-toolchain/toolchain-e2e/pull/675/commits/f022835dee599e36bcb642f96b8ff69428ffe463)

Thanks for pointing that out!